docs/ShippingSimulator.md

```diff
@@ -40,11 +40,9 @@ You can use it in your code like a React component with the jsx tag: `<ShippingS
 
 You should follow the Styles API instruction in the main [README](/README.md#styles-api).
 
-#### CSS Namespaces
+#### Customization
 
-Below, we describe the namespace that are defined in the `ShippingSimulator`.
-
-| Class name                   | Description                                                | Component Source                                                                       |
+| CSS Handle                   | Description                                                | Component Source                                                                       |
```

Comment:
Remove the last column

Reply:
You could also only keep the `CSS Handle` column